src/app/fdctl/run/tiles/fd_shred.c

```diff
@@ -460,7 +420,6 @@ send_shred( fd_shred_ctx_t *      ctx,
 
   memcpy( hdr->eth->dst, dest->mac_addr, 6UL );
 
-  memcpy( hdr->ip4->daddr_c, &dest->ip4, 4UL );
   hdr->ip4->net_id     = fd_ushort_bswap( ctx->net_id++ );
   hdr->ip4->check      = 0U;
   hdr->ip4->check      = fd_ip4_hdr_check( ( fd_ip4_hdr_t const *) FD_ADDRESS_OF_PACKED_MEMBER( hdr->ip4 ) );
```

Comment:
Don't need to calculate checksum here? It's going to be wrong anyway right?

Reply:
no - the memcpy is necessary - it shouldn't have been removed